### PR TITLE
Stats schema update/20240402/v2

### PR DESCRIPTION
**Pull request by inashivb**
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket: https://redmine.openinfosecfoundation.org/issues/6434

Previous PR: https://github.com/OISF/suricata/pull/10749

Changes since v1:
- rebased

### CODE REVIEW

File: etc/schema.json [hunk 1]
```diff
@@ -4181,63 +4220,96 @@
...
                         "rejected": {
+                            "description": "Number of rejected packets",
                             "type": "integer"
                         },
                         "replaced": {
+                            "description": "Number of replaced packets",
                             "type": "integer"
                         },
                         "drop_reason": {
+                            "description": "Number of dropped packets per reason",
```
Comment (jufajardini) [inline etc/schema.json:4239]: nit:
```suggestion
                            "description": "Number of dropped packets, grouped by drop reason",
```

File: etc/schema.json [hunk 2]
```diff
@@ -4181,63 +4220,96 @@
...
+                                            "Number of packets dropped due to flow memcap exception policy",
                                     "type": "integer"
                                 },
                                 "flow_drop": {
+                                    "description": "Number of packets dropped due to dropped flows",
                                     "type": "integer"
                                 },
                                 "applayer_error": {
+                                    "description":
+                                            "Number of packets dropped due to global app-layer error policy",
```
Comment (jufajardini) [inline etc/schema.json:4268]: This one is also due to exception policies, so I suggest following the same style:
"Number of packets dropped due to app-layer error exception policy"

**Review by jufajardini — CHANGES_REQUESTED** [review]
This is a nice work, thank you :)

File: etc/schema.json (hunk 2 above)
Comment (inashivb) [reply etc/schema.json:4268]: Thank you! :) I explicitly requested review from you bc these docs were about exception policies. Knew you'd find something. 😛

**Pull request closed without merging**